app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/details/DetailViewExampleActivity.java

```diff
@@ -24,9 +24,27 @@
  */
 public class DetailViewExampleActivity extends Activity {
 
+    /***
+     * True if using trailer as a background.
+     */
+    protected boolean hasBackgroundVideo() {
```

Comment:
can you create a new activity and a clone a new Fragment instead of subclass and using parameters?
We need show the concept to developer, so make code as straightforward as possible.

You will need two video urls in DetailsCard object, one for trailer, one for main video.